db/app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const {getAllTopics} = require("../controller/controller")
+const {handlePsqErrors, handleCustomErrors,handle404} = require("../error")
+
+const app = express();
+app.use(express.json())
+
+app.get('/api/topics', getAllTopics)
+
+
+
+
+app.use(handle404)
+
+app.use(handlePsqErrors)
```

Comment:
I'm not sure why you have this at the moment as you're only testing for a bad path.